.github/workflows/unittests.yml

```diff
@@ -70,12 +70,16 @@ jobs:
       - name: Setup docker container
         run: |
           EXTRA_DOCKER_OPTIONS="--name velox-backend-test-$GITHUB_RUN_ID --detach" NON_INTERACTIVE=ON gluten-te/cbash.sh sleep 14400
-      - name: Install Gluten Velox-backend spark3.2 and Unit test
+      - name: Install Gluten Velox-backend spark3.2 and Build Unit test
```

Comment:
Not just build, mvn without -DskipTests, it will run the tests

Reply:
Thanks. Updated.